Firestore/Source/Remote/FSTDatastore.m

```diff
@@ -522,14 +572,16 @@ - (void)resumeStartWithToken:(FSTGetTokenResult *)token error:(NSError *)error {
   [FSTDatastore prepareHeadersForRPC:_rpc
                           databaseID:self.databaseInfo.databaseID
                                token:token.token];
-  [_rpc startWithWriteable:self];
+  FSTAssert(_grxFilter == nil, @"GRX Filter must be nil");
+  _grxFilter = [[GRXFilter alloc] initWithStream:self];
```

Comment:
Again, the "grx" prefix is something we should avoid. Maybe `_callbackFilter`?

Reply:
Renamed to _callbackFilter to match FSTCallbackFilter.